js-tasks/task1.js

```diff
@@ -0,0 +1,49 @@
+function summ() {
```

Comment:
Add check for existence (if undefined -> return) to avoid errors

(*for every function)